src/main/java/mate/academy/spring/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.request;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderRequestDto {
+    private List<Long> ticketIds;
+    private LocalDateTime orderDate;
+    private Long userId;
```

Comment:
Need we return userId if frontender ask the order by the id (they know this id)?